src/pages/HomePage/PricingCard.jsx

```diff
@@ -1,43 +1,130 @@
 import React from 'react';
-import { Card, CardContent, Typography, Button } from '@mui/material';
-
-function PricingCard({ title, price, description, benefits, buttonText, highlight = false }) {
-    const listItemStyles = {
-        fontSize: '0.9rem',
-        fontWeight: 'normal',
-        lineHeight: '1.5',
-        color: 'white',
-    };
+import { Box, Typography, Button } from '@mui/material';
 
+function PricingCard({ title, price, description, benefits, buttonText }) {
```

Comment:
remove unused props ```title, price, description```

Reply:
@AlanisYates The unused props have been removed